### DISCUSSION TITLE
setting the endpoint url by 'graphql.servlet.mapping' not working

### DISCUSSION THREAD

**Discussion by menilub** [graphql-java-kickstart/graphql-spring-boot#448] [Q&A]
if you try setting the graphql endpoint url by:
`graphql.servlet.mapping=`
not working

`graphql.url =`
works

need to fix the documentation. found the solution in bug #107

**Comment by oliemansm** [comment]
Looking at the code the `graphql-kickstart-spring-boot-starter-webflux` is using the `graphql.url` property. Whereas `graphql-spring-boot-starter` is using `graphql.servlet.mapping`.

Can you confirm you were in fact facing this issue using the Webflux starter? You can mark this comment as the correct answer in that case.
